manifests/o/Orange-OpenSource/Hurl/2.0.1/Orange-OpenSource.Hurl.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Orange-OpenSource.Hurl
+PackageVersion: 2.0.1
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+Installers:

```

Comment:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

Reply:
Hi @lepapareil,

Sorry to forget the label yesterday, as the accompanying message is somewhat informative and may have answered your question. The root issue is that Microsoft.VCRedist.2015+.x64 is required for this package to execute, but is not included on all systems. Our validations are performed on fresh installs of Windows, and ours do not have these files preinstalled.

Once dependencies are fully supported, this suggestion should have WinGet install this dependency before installing the Orange main package.

Reply:
Many thks @stephengillie :)

I pushed your suggestion https://github.com/lepapareil/winget-pkgs/commit/441e052c1b2307f7bab66a756f3b985ca6dd4131